test/e2e/conftest.py

```
@@ -124,3 +131,138 @@ def pytest_addoption(parser):
+    ("cluster_stats", "/stats?filter=cluster_manager"),
+    ("server_memory", "/stats?filter=server.memory"),
+    ("wasm_stats", "/stats?usedonly&filter=wasm"),
+]
+
+_SNAPSHOT_INTERVAL = 300
+
+
+def _find_running_pod(cli, namespace, name_prefix):
+    """Find a running pod whose name starts with name_prefix."""
+    try:
+        result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)